SWRevealViewController/SWRevealViewController.m

```diff
@@ -1023,16 +1028,16 @@ - (void)_notifyPanGestureEnded
 
 - (void)_getRevealWidth:(CGFloat*)pRevealWidth revealOverDraw:(CGFloat*)pRevealOverdraw forSymetry:(int)symetry
 {
-    if ( symetry < 0 ) *pRevealWidth = _rightViewRevealWidth, *pRevealOverdraw = _rightViewRevealOverdraw;
-    else *pRevealWidth = _rearViewRevealWidth, *pRevealOverdraw = _rearViewRevealOverdraw;
+    if ( symetry < 0 ) (void)(*pRevealWidth = _rightViewRevealWidth), *pRevealOverdraw = _rightViewRevealOverdraw;
```

Comment:
This is not the normal way to write this code. Just at braces and semicolons.
if ( symetry < 0 ) {*pRevealWidth = _rightViewRevealWidth; *pRevealOverdraw = _rightViewRevealOverdraw; }
else {*pRevealWidth = _rearViewRevealWidth; *pRevealOverdraw = _rearViewRevealOverdraw; }